components/footer.js

```diff
@@ -1,14 +1,15 @@
-import React from "react";
-import { Download as DownloadIcon } from "lucide-react";
 import Dropzone from "components/dropzone";
-import { XCircle as StartOverIcon } from "lucide-react";
-import { Code as CodeIcon } from "lucide-react";
-import { Info as InfoIcon } from "lucide-react";
+import {
+  Code as CodeIcon,
+  Download as DownloadIcon,
+  Info as InfoIcon,
+  XCircle as StartOverIcon,
+} from "lucide-react";
 import Link from "next/link";
 
 export default function Footer({ events, startOver, handleImageDropped }) {
   return (
-    <div className="w-full">
+    <footer className="w-full">
```

Comment:
The footer could use some breathing room up top while the first prediction is running.

 
![Screenshot 2023-01-29 at 6 43 15 PM](https://user-images.githubusercontent.com/2289/215376582-ec69da9c-e3b4-45f8-aec1-dca4116e95f6.png)